src/test/java/com/amazonaws/services/dynamodbv2/datamodeling/encryption/DynamoDBEncryptorTest.java

```diff
@@ -58,6 +59,7 @@
 import com.amazonaws.services.dynamodbv2.datamodeling.internal.Utils;
 import com.amazonaws.services.dynamodbv2.model.AttributeValue;
 import com.amazonaws.services.dynamodbv2.testing.AttrMatcher;
+import org.w3c.dom.Attr;
```

Comment:
Is this being used anywhere? I can't find it. Perhaps IDE added this when you typed AttributeValue?